Fix minor input issues (#25)
obi1kenobi/cargo-semver-checks-action: README.md

| Input              | Description                                                                                                                       | Default |
|--------------------|-----------------------------------------------------------------------------------------------------------------------------------|---------|

> | `package`            | The package whose API to check for semver (in Package Id Specification format, see https://doc.rust-lang.org/cargo/reference/pkgid-spec.html for reference). If not set, all packages in the workspace are processed. | |

Reviewer: ```suggestion
| `package`            | The package whose API to check for semver (in Package Id Specification format, see https://doc.rust-lang.org/cargo/reference/pkgid-spec.html for reference). If not set, all packages defined in the Cargo.toml file are processed. | |
```
Tweak because if the `manifest-path` points to a crate manifest within a workspace, then only that crate will be checked even if `package` isn't set.

Author: You're right, I missed this case

> | `manifest-path`      | Path to Cargo.toml of crate or workspace to check. By default, the action will assume it exists in the current directory. | |

Reviewer: "Current directory" is a tricky concept since we're talking about a GitHub Action invoked from a YAML file, not via the shell.

Perhaps:
```suggestion
| `manifest-path`      | Path to Cargo.toml of crate or workspace to check. By default, the action assumes the manifest is at the repo root. | |
```

Author: Well, current directory might not be very clear indeed, but is "repo root" really better? If I've read "repo root", I'd ask myself "which repo?" Not the repository checked for semver violations, as it might be cloned somewhere else, and not the repository with the action... In fact at the beginning there is no repo. What we mean here is some kind of "current directory", but in terms of the context in which the action is run.

Reviewer: Hmm, good point.

How about "default GitHub Actions directory for the project" or something to that effect?

Author: After a bit of googling I found out about `GITHUB_WORKSPACE` variable, which according to https://docs.github.com/en/actions/learn-github-actions/variables#default-environment-variables is "The default working directory on the runner for steps, and the default location of your repository when using the [checkout](https://github.com/actions/checkout) action." which is exactly what we want to describe.

What do you think about saying that the action "assumes the manifest is under `GITHUB_WORKSPACE` path" and linking https://docs.github.com/en/actions/learn-github-actions/variables#default-environment-variables? Or maybe just saying that it is "the default working directory on the runner for steps" is better?

Reviewer: I like "assumes the manifest is under the default `GITHUB_WORKSPACE` path" + the link you found. Ship it 🚀

Author: Changed!
